substrate-node/runtime/src/lib.rs

```diff
@@ -511,7 +511,7 @@ impl sygma_bridge::Config for Runtime {
 	type FeeReserveAccount = TreasuryAccount;
 	type DestChainID = DestChainID;
 	type DestVerifyingContractAddress = DestVerifyingContractAddress;
-	type FeeHandler = sygma_basic_feehandler::BasicFeeHandlerImpl<Runtime>;
+	type FeeHandler = SygmaBasicFeeHandler;
```

Comment:
should here be the FeeHandlerRouter instead of a concrete FeeHandler?

Reply:
For sure, I only configured it in router mock runtime for demo purposes. If we wanna use a fee router, we definitely can change it to a fee router.